### PR TITLE
Etca playbooks 1

### PR DESCRIPTION
**Pull request by cat-bro**

### CODE REVIEW

File: hosts [hunk 1]
```diff
@@ -198,6 +198,33 @@ cvmfsp-paw.gvl.org.au ansible_ssh_host=146.118.70.86 internal_ip=192.168.0.131
 [galactic_radio_telescope]
 telescope ansible_ssh_host=115.146.86.144
 
+[galaxy_group]
+galaxy ansible_ssh_host=138.44.7.169 internal_ip=192.168.210.157
+galaxy-handlers ansible_ssh_host=138.44.7.173 internal_ip=192.168.210.252
+galaxy-db ansible_ssh_host=138.44.7.171 internal_ip=192.168.210.128
+galaxy-queue ansible_ssh_host=138.44.7.170 internal_ip=192.168.210.230
+galaxy-backup ansible_ssh_host=138.44.7.172 internal_ip=192.168.210.239
+[galaxy_group:children]
```
Comment (nuwang) [inline hosts:207]:
```suggestion

[galaxy_group:children]
```

File: galaxy-backup_playbook.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  roles:
+    # - galaxyproject.repos # probably not needed for this VM
+    # - common
+    - geerlingguy.pip
+    # - slg.db-backup # Do not include this role until galaxy production is in Sydney
+    # - dj-wasabi.telegraf # comment this out for now - need to check which db it sends to, and that the db exists on stats
+  post_tasks:
+    - name: Ensure desired version of ansible is installed
+      pip:
+        name: "ansible==4.0.0a1"
```
Comment (nuwang) [inline galaxy-backup_playbook.yml:25]: This is a pre-release version. Maybe 4.0.0? Also, it's pretty old, current version id 8.0.0

Comment (cat-bro) [reply galaxy-backup_playbook.yml:25]: 4.0.0 would be OK too. The current version is too new.

**Comment by cat-bro** [comment]
Things to add:

gx_it_proxy change
postfix-mail-relay role

**Pull request merged**